docs/source/sections/gcp-info/GeneFlow-RNAseq.rst

```diff
@@ -0,0 +1,225 @@
+	      output: ${1}.sam
+
+	  sort:
+	    app: bam-sort
+	    depend: [ "align" ]
+	    map:
+	      uri: ${align->output}
+	      regex: (.*).sam
+	    template:
+	      input: ${align->output}/${1}.sam
+	      output: ${1}.bam
+
+	  quantify:
+	    app: stringtie
+	    depend: [ "sort" ]
+	    map:
+	      uri: ${sort->output}
+	      regex: (.*).bam
+	    template:
+	      bam: ${sort->output}/${1}.bam
+	      gtf: ${workflow->gtf}
+	      output: ${1}
+	...
+
+The **inputs** and **parameters** blocks define the inputs and paramaters that need to be passed to the workflow upon execution. Some of these inputs and parameters are optional or have default values. 
+
+The **apps** block lists all apps used by the workflow and links to other, reusable source repositories for each app. Learn more about how each app works by following the Git repository links below:
+
+* `HISAT2 Build <https://gitlab.com/geneflow/apps/hisat2-build-gf2.git>`_
+* `HISAT2 Align <https://gitlab.com/geneflow/apps/hisat2-align-gf2.git>`_
+* `BAM Sort <https://gitlab.com/geneflow/apps/bam-sort-gf2.git>`_
+* `StringTie <https://gitlab.com/geneflow/apps/stringtie-gf2.git>`_
+
+The **steps** block defines the order of app execution as well as step dependencies for each app. It also defines how apps are chained together via their inputs and outputs. 
+
+To run the workflow:
+
+::
+
+	# assuming the GeneFlow Python virtual environment has been activated, view the command line help
+ 	$ gf help GeneFlow-RNAseq
+
+	# run the workflow
+	$ cd GeneFlow-RNAseq
+	$ gf run . -o output 
+
+After the workflow completes, the output folder should look similar to this:
+
+::
+
+	output
+	└── geneflow-job-095ba2fe
+	    ├── quantify
+	    │   ├── _log
+	    │   │   ├── gf-0-quantify-sample.err
+	    │   │   ├── gf-0-quantify-sample.out
+	    │   │   ├── sample-stringtie.stderr
+	    │   │   └── sample-stringtie.stdout
+	    │   └── sample
+	    │       ├── e2t.ctab
+	    │       ├── e_data.ctab
+	    │       ├── i2t.ctab
+	    │       ├── i_data.ctab
+	    │       ├── sample.tsv
+	    │       ├── sample_final_reference.gtf
+	    │       ├── sample_final_transcript.gtf
+	    │       └── t_data.ctab
+	    └── sort
+		├── _log
+		│   ├── gf-0-sort-sample-bam.err
+		│   ├── gf-0-sort-sample-bam.out
+		│   └── sample.bam-samtools-sort.stderr
+		└── sample.bam
+
+The script will run Docker containers for `hisat2 <http://daehwankimlab.github.io/hisat2/>`_ , `samtools <http://www.htslib.org/>`_, and `stringtie <https://ccb.jhu.edu/software/stringtie/>`_ to do the work. **sample.bam** contains the sequence alignment data produced by mapping reads to the reference genome **sample.bam**. Additional information about gtf and tsv outputs of stringtie can be found `here <http://ccb.jhu.edu/software/stringtie/index.shtml?t=manual>`_. The **sample_final_transcript.gtf** contains details of the transcripts that StringTie assembles from RNA-Seq data, while **sample.tsv** contains gene abundances.
+

```

Comment:
There doesn't appear to be any new results in here.

Reply:
The main result files should be the same as the other RNAseq workflows.